src/fiatExchanges/FiatExchangeAmount.tsx

```diff
@@ -157,20 +143,20 @@ function FiatExchangeAmount({ route }: Props) {
       dispatch(
         attemptReturnUserFlow({
           flow,
-          selectedCrypto: currency,
+          selectedCrypto: resolveCICOCurrency(tokenSymbol),
```

Comment:
In this case (as well as when a similar thing is done in `SelectProviders.tsx`) I am not quite sure what to do. This is part of the fiat connect flow, which the ticket says is out of scope for this issue. In looking through the code I'm not sure how to avoid this without also reworking all of the fiat connect quote fetching, is that what I should do?